_hooks/pre-commit

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+
+# A git hook script that runs the same checks that are 
+# executed on Travis to verify your commits will not fail.
+# Bypass it with the --no-verify option to git-commit
+#
+# usage: make a symlink to this file,
+# e.g., rm -rf .git/hooks/ && ln -s -f ../_hooks .git/hooks
+
+# Run 
+./_tests/travis-checks.sh --disable-external
+
+if [ $? -eq 0 ]; then
+    exit 0
+else
+  echo "**********************************************************"
+  echo -e "*  \033[31mCOMMIT ABORTED! Please address the errors before commiting.\033[0m"
```

Comment:
I appreciate the nice formatting, but I think it makes the code harder to maintain. I'd prefer to just have a simple error message with no colored fonts or star separators.